specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,408 @@
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+              "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+              "name": "resourceNamespace",
+              "in": "path",
+              "description": "The Namespace of the resource.",
+              "required": true,
+              "type": "string"
+          },
+          {
+              "name": "resourceType",
+              "in": "path",
+              "description": "The type of the resource.",
+              "required": true,
+              "type": "string"
+          },
+          {
+              "name": "resourceName",
+              "in": "path",
+              "description": "Name of the resource.",
+              "required": true,
+              "type": "string"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/NetworkData"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkDataList": {
+      "type": "object",
+      "description": "List of network data",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkData"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "NetworkData": {
+      "type": "object",
+      "description": "Network data on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkDataProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "NetworkDataProperties": {
+      "type": "object",
+      "description": "Describes the properties of the network data.",
+      "properties": {
+        "connectableResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkDataConnectableResource"
+          }
+        }
+      }
+    },
+    "NetworkDataConnectableResource": {
+      "type": "object",
+      "description": "Describes the allowed inbound and outbound traffic of an Azure resource",
+      "properties": {
+        "connectedResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connectable resource id"
```

Comment:
connectable or connected?
also, try to have meaningful description as possible
if users do not understand the field from its name and go to the documentation we want to help them

Reply:
for all here :)
currently I have to go to the examples and hopefully understand some of the fields